app/src/main/java/com/quran/labs/androidquran/ui/util/TranslationsSpinnerAdapter.java

```diff
@@ -36,6 +37,9 @@ public TranslationsSpinnerAdapter(Context context,
     // and the String[] constructor makes a new (immutable) List with the items of the array.
     super(context, resource, new ArrayList<>());
     this.layoutInflater = LayoutInflater.from(context);
+    List<String> transList = new ArrayList<String>(Arrays.asList(translationNames));
+    transList.add("More Translations");
+    translationNames = transList.toArray(new String[transList.size()]);
```

Comment:
you have two options - one is to add it within the underlying array, and the other is to change the adapter to know that "count is always size + 1," and to know that "once you pass this element, the next element is the more option" - if it's cleaner to add it to the array (i.e. if the array is all Strings, as it seems to be), then it probably makes sense.